lib/src/providers/TranslationProvider/index.tsx

```diff
@@ -1,23 +1,24 @@
 import { PropsWithChildren, useMemo } from 'react';
-import { createIntl, createIntlCache, RawIntlProvider, type ResolvedIntlConfig } from 'react-intl';
+import { createIntl, createIntlCache, RawIntlProvider } from 'react-intl';
+import { IntlShape } from 'react-intl/src/types';
 import { VERNA_SUPPORTED_LOCALES } from ':/settings';
 import { TranslationType } from ':/types/translations';
 import resourceLoader, { type ResourceLoader } from ':/utils/suspense/resourceLoader';
 
 interface TranslationProviderProps {
   defaultLocale?: string;
-  intl?: ResolvedIntlConfig;
+  intl?: IntlShape;
   locale?: string;
   schemaTranslations?: TranslationType;
 }
 
-type MessageLoader = ResourceLoader<ResolvedIntlConfig['messages']>;
+type MessageLoader = ResourceLoader<IntlShape['messages']>;
 type MessageLoaders = Record<string, MessageLoader>;
 
 const messagesLoader = (locale: string): MessageLoader =>
   resourceLoader(() =>
     import(`../../translations/${locale}.json`).then(
-      (response: { default: ResolvedIntlConfig['messages'] }) => response.default,
+      (response: { default: IntlShape['messages'] }) => response.default,
```

Comment:
Why do you need to change this type ? 
I prefer to import a type exposed by the main module instead of import a type from source code.

Reply:
Hey it was an old way of doing things that i removed because it caused problems, i needed something only inside IntlShape (which extends ResolveIntlconfig), i'll remove it, well seen

Reply:
If you trully don't like it i can override it with a custom type formatMessage & locale